docs/understanding/history-and-design/PriorArt.md

```diff
@@ -27,11 +27,11 @@ For what it's worth, Flux's creators [approve](https://twitter.com/jingc/status/
 
 Unlike Redux, Elm is a language, so it is able to benefit from many things like enforced purity, static typing, out of the box immutability, and pattern matching (using the `case` expression). Even if you don't plan to use Elm, you should read about the Elm architecture, and play with it. There is an interesting [JavaScript library playground implementing similar ideas](https://github.com/paldepind/noname-functional-frontend-framework). We should look there for inspiration on Redux! One way that we can get closer to the static typing of Elm is by [using a gradual typing solution like Flow](https://github.com/reduxjs/redux/issues/290).
 
-### Immutable
+### Immer
 
-[Immutable](https://facebook.github.io/immutable-js) is a JavaScript library implementing persistent data structures. It is performant and has an idiomatic JavaScript API.
+[Immer](https://immerjs.github.io/immer/) is a JavaScript library implementing persistent data structures. It is performant and has an idiomatic JavaScript API. Immer operates on normal JavaScript objects and arrays.
```

Comment:
:hand: Immer does _not_ use "persistent data structures".  

I would actually say let's revert the changes to this paragraph entirely, because Redux _was_ inspired by Immutable.js